src/util.ts

```diff
@@ -250,26 +250,30 @@ function querySideloadingRegistry(commands: string[]): Promise<string> {
   });
 }
 
-function addManifestToRegistry(manifestId: string, manifestPath: string): Promise<any> {
-  return querySideloadingRegistry(['Set-ItemProperty -LiteralPath $RegistryPath -Name "' + manifestId + '" -Value "' + manifestPath + '"']);
+async function addManifestToRegistry(manifestId: string, manifestPath: string): Promise<any> {
+  await querySideloadingRegistry(['Remove-ItemProperty -LiteralPath $RegistryPath -Name "' + manifestPath + '" -ErrorAction SilentlyContinue']);
```

Comment:
Please add a comment that explains why it needs to remove the manifest path when adding the manifest id.